src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -1552,4 +1552,147 @@ public void testGetExerciseStatsForCourseOverviewWithPastExercises() throws Exce
         var statisticsOptional = exerciseDTOS.stream().filter(exercise -> exercise.getExerciseId().equals(exerciseId)).findFirst();
         assertThat(statisticsOptional.isEmpty()).isTrue();
     }
+
+    // Test
```

Comment:
@Test seems to be missing

Reply:
The methods in this file are not annotated with `@Test` since they are helper methods called by the GitlabJenkins and BambooJira tests.

Reply:
Why do we need such an annotation? I don't see any usage of this in this test class